packages/file_selector/file_selector_web/README.md

```diff
@@ -18,8 +18,8 @@ This is what the above means to your `pubspec.yaml`:
 ...
 dependencies:
   ...
-  file_picker: ^0.7.0
-  file_picker_web: ^0.7.0
+  file_picker: ^0.7.0+2
+  file_picker_web: ^0.7.0+1
```

Comment:
The problem is the name being wrong, not the exact versions, so this doesn't actually fix the issue.

Reply:
Updated the branch with the correct fix, PTAL :)